src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -147,6 +147,7 @@ internal enum MessageID
         IDS_FeaturePrivateProtected = MessageBase + 12730,
 
         IDS_FeatureAttributesOnBackingFields = MessageBase + 12731,
+        IDS_FeatureRange = MessageBase + 12732,
```

Comment:
Consider using 12733, as I have a PR using 12732 (for tuple equality feature). That will simplify merge conflict. Thanks

[https://github.com/dotnet/roslyn/pull/23356/files#diff-a672b28d02b502f46551f7ece1c4c5d8](https://github.com/dotnet/roslyn/pull/23356/files#diff-a672b28d02b502f46551f7ece1c4c5d8)